pkg/kube/template.go

```diff
@@ -43,13 +44,23 @@ func NewTemplate(template map[string]interface{}, backend types.Backend, prefix
 		kvVersion = kv
 	}
 
-	// Only worry about getting Vault secrets for templates with <placeholder>'s
-	replaceable := replaceableInner(&template)
-	var data map[string]interface{}
-	if replaceable {
-		data, err = backend.GetSecrets(path, kvVersion)
+	var avpIgnore bool
```

Comment:
I think we can set this to false on initialization on not worry about the err in converting the string to bool

Reply:
Actually bool default is false so really I can just ignore the `err`